nautobot/dcim/fields.py

```diff
@@ -70,7 +70,7 @@ def python_type(self):
         return EUI
 
     @property
-    def validators(self):
+    def validators(self):  # pylint: disable=invalid-overridden-method
```

Comment:
Should we change this to a `@cached_property` decorator like the parent class?

Reply:
Yeah, this looks like less-than-ideal code. I'll open a child PR with what I think should be the correct code. Good spot!

Reply:
#2332.